app/controllers/tasks_controller.rb

```diff
@@ -31,16 +37,35 @@ def create
   def destroy
     @task = Task.find(params[:id])
     @task.destroy
+    @task.save #this might not be necessary
+
```

Comment:
I don't think you need the save after the destroy, some doc confirmation here: http://apidock.com/rails/ActiveRecord/Relation/destroy